examples/js/test-js.js

```diff
@@ -22,16 +22,32 @@ const cppCodeBlock = `#include <iostream>
 int main()
 {
   auto curried_add = [](int x) -> function<int(int)> { return [=](int y) { return x + y; }; };
-  
+
   auto answer = curried_add(7)(8);
   std::cout << answer << std::endl;
-  
+
   return 0;
 }`;
 
 const TestJs = () => (
-  <Deck>
-    <Slide slideNum={1}>
```

Comment:
We no longer need these as we add them via props based on their order in the tree.